### ISSUE TITLE
Error on pip install for Windows 10

### ISSUE THREAD

**Issue by TousakaJun** [pmrowla/pylivemaker#115]
* pylivemaker version: 1.0.4
* Python version: 3.12
* Operating System: Windows 10

### Description

While trying to `py -m pip install pylivemaker` I get installation error:
```
File "C:\Users\User\AppData\Local\Temp\pip-install-7ywisi7y\construct_985a2c31eb0f4bfdaee9783020fe99e7\construct\core.py", line 3, in <module>
          import struct, io, binascii, itertools, collections, pickle, sys, os, tempfile, hashlib, importlib, imp
      ModuleNotFoundError: No module named 'imp'
```

Today I tryed the same installation process on Linux and it worked properly. Can you make any suggestions on how to fix this Windows installation problem?

### What I Did

I've tryed to just copy livemaker to the python lib folder, and install `lxml` and `construct`.
First problem was that `construct` package requares 2.8.12 version, because newer versions has no `construct.Embedded`.
After 2.8.12 version was installed, I get this error:
```
File "C:\Users\User\AppData\Local\Programs\Python\Python312\Lib\site-packages\construct\core.py", line 1884, in __init__
    subcon, value = Bytes(len(value)), value
TypeError: object of type 'Mapping' has no len()
```
It's obvious that `pylivemaker` should be on Python 3.8+, still this `has no len()` problem is mostly for map-objects and can be solved by running a Python 2. So now I'm confused and don't know how to fix this problem.

**Comment by pmrowla** [comment]
`imp` was deprecated and removed in python 3.12, on windows if you switch to python 3.11 it should work as expected.

The latest pylivemaker from source should also work in 3.12 on windows, since it has been updated to support the current construct release, you can install from source with
```
pip install git+https://github.com/pmrowla/pylivemaker.git
```

**Comment by pmrowla** [comment]
`pylivemaker==1.1.0` is now released that should work with python 3.12 on windows You should be able to just update your existing installation with `pip install -U pylivemaker`
